src/python/org_fetcher/fetch_orgs.py

```diff
@@ -0,0 +1,64 @@
+from collections import defaultdict
+import csv
+import os
+
+PATH_TO_DATAFILES = os.path.join(
+    '..', '..', 'php', 'gov-harvester', 'datafiles'
+)
+
+
+orgs = defaultdict(lambda: 0)
+for filename in os.listdir(PATH_TO_DATAFILES):
+    filename_lower = filename.lower()
+    if not filename_lower.endswith('.csv'):
+        # Skip the file if it's not a csv
+        continue
+    if not 'meetings' in filename_lower:
+        # Skip the file if it doesn't contain a meeting of some sort
+        continue
+    if 'gift' in filename_lower:
```

Comment:
Same as above 😄 Maybe `contains_gift(filename_lower)`?

Come to think of it, we could take this a step further and a function called `should_skip(filename)` that just calls the three little functions from above:
```
function should_skip($filename) {
  return is_csv($filename) || contains_meetings($filename) || contains_gift($filename)
}
```

That method explains when we skip entire files 😸